forge-gui/res/cardsfolder/upcoming/maximum_carnage.txt

```diff
@@ -0,0 +1,10 @@
+Name:Maximum Carnage
+ManaCost:4 R
+Types:Enchantment Saga
+K:Chapter:3:DBEffect,DBMana,DBDamage
+SVar:DBEffect:DB$ Effect | Duration$ UntilYourNextTurn | StaticAbilities$ MustAttack1,MustAttack2 | SpellDescription$ Until your next turn, each creature attacks each combat if able and attacks a player other than you if able.
+SVar:MustAttack1:Mode$ MustAttack | ValidCreature$ Creature | Description$ Until your next turn, each creature attacks each combat if able and attacks a player other than you if able.
+SVar:MustAttack2:Mode$ MustAttack | ValidCreature$ Creature | MustAttack$ Player.Other | Secondary$ True | Description$ Until your next turn, each creature attacks each combat if able and attacks a player other than you if able.
```

Comment:
@tool4ever does this work?
If yes, can "Kardur, Doomscourge" be updated too?

Reply:
good question, I will investigate